src/ansys/tools/report.py

```diff
@@ -147,13 +147,21 @@ def project_info(self):
                     env_info_lines.append(f"{key:<30} {value}")
                     n_var += 1
 
-        # Now, take into account the default env variables
-        for ansys_default in __ANSYS_VARS__:
-            for key, value in os.environ.items():
-                if (ansys_default in key) and (key not in self._ansys_vars):
-                    env_info_lines.append(f"{key:<30} {value}")
-                    n_var += 1
-
+        # Loop over all environment variables
+        for key, value in os.environ.items():
+            # Now, take into account the default env variables
+            for ansys_default in __ANSYS_VARS__:
+                # Check if the "ansys_default" substring is found
+                if ansys_default in key:
```

Comment:
For making sure that it an Ansys variable, use the `startswith` method:

```suggestion
                if key.startswith(ansys_var_prefix):
```

Otherwise, variables like `MALICIOUS_FUENT_VAR` would be detected as Ansys variables.

Reply:
Also, you may also consider creating a function just for checking if a variable is an Ansys one:

```python
def _is_ansys_var(env_var):
    for prefix in __ANSYS_VARS__:
        if env_var.startswith(prefix + "_"):
            return True
```

This way indentation level can be reduced.

Reply:
Thanks for the function suggestion, I do agree on that.

And agreed on the startswith. I wanted to leave space for those variables which have the keyword somewhere in the middle. So, it was done on purpose the use of "key in X" instead of using startswith. But I'll change it. Honestly, I think all of them will start with those prefixes